templates/commands/render/flags.go

```diff
@@ -91,6 +94,14 @@ func (r *RenderFlags) Register(set *cli.FlagSet) {
 		Usage:   "The key=val pairs of template values; may be repeated.",
 	})
 
+	f.StringVar(&cli.StringVar{
+		Name:    "input-file",
+		Example: "/my/git/abc-inputs.yaml",
+		Default: "",
+		Target:  &r.InputFile,
+		Usage:   "The file with the key=val pairs of template values.",
```

Comment:
It might be misleading to say "key=val" pairs since it's a YAML file

Reply:
Done